### PR TITLE
Solo anc stix ephemeris

### PR DESCRIPTION
**Pull request by nicHoch**
rename L2-aux files to ANC soar schema 

https://github.com/i4Ds/STIXCore/issues/396

### CODE REVIEW

File: stixcore/processing/publish.py
Comment (samaloney) [inline stixcore/processing/publish.py:547]: asp = aspect or should it be anc = ancialliar?

**Review by samaloney — COMMENTED** [review]
LGTM would need to be updated once #393 is merged

File: stixcore/processing/publish.py
Comment (nicHoch) [reply stixcore/processing/publish.py:547]: yes: asp = aspect 
ANC is allready in the level
and i added a new type as it is not HK anymore

**Review by samaloney — APPROVED** [review]
LGTM

**Pull request merged**
